.azure/pipelines/components-e2e-tests.yml

```diff
@@ -44,9 +44,15 @@ jobs:
       displayName: NPM install
     - script: .dotnet/dotnet build ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-restore
       displayName: Build
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter QuarantinedTest\!~github --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined!=true|Quarantined=false'
```

Comment:
I accidentally dropped `-p:RunQuarantinedTests=true` in my suggestion I think.

Reply:
We have this on line 52 for the quarantined runs. I don't believe we need that switch for the non-quarantine run on line 47 here.

Reply:
My bad